book.toml

```
@@ -1,5 +1,6 @@
 [book]
 authors = ["Ankitects Pty Ltd and contributors"]
+description = "The official Manual for Anki, the flashcard program that makes learning easier."
```

`Manual`'s captitalization is odd.

How about this instead:

```suggestion
description = "Anki's user manual. Anki is a flashcard program that makes learning easier."
```